src/components/dropdown/dropdown.tsx

```diff
@@ -34,7 +43,7 @@ import { createObserver } from "../../utils/observers";
   styleUrl: "dropdown.scss",
   shadow: true
 })
-export class Dropdown {
+export class Dropdown implements ConditionalSlotComponent {
```

Comment:
Could maybe have a interface to implement `ReferenceElementComponent`?

Reply:
We could also add a `referenceElement` property as part of `FloatingUIComponent`

https://github.com/Esri/calcite-components/pull/3836/files#diff-3c419c81de1cc684c4d1d859881782bc87d089da410675b5af3f6c468b95ef13R43